plenum/common/ledger_manager.py

```diff
@@ -112,24 +112,35 @@ def request_CPs_if_needed(self, ledgerId):
         ledgerInfo.consistencyProofsTimer = None
         ledgerInfo.recvdCatchupRepliesFrm = {}
 
+    @staticmethod
+    def _missing_txns(ledger_info) -> Tuple[bool, int]:
+        ledger = ledger_info.ledger
+        if ledger_info.catchupReplyTimer is None:
+            return False, 0
+
+        end = getattr(ledger_info.catchUpTill, f.SEQ_NO_END.nm)
+
+        catchUpReplies = ledger_info.receivedCatchUpReplies
+        total_missing = (end - ledger.size) - len(catchUpReplies)
```

Comment:
`ledger_info.receivedCatchUpReplies` is only updated with txns with seq_no > ledger.size, have a look at `processCatchupRep`